modules/402-ingress-nginx/templates/controller/admission.yaml

```diff
@@ -34,6 +40,39 @@ webhooks:
     {{- if semverCompare "<1.22" $kubernetesVersion}}
       - v1beta1
     {{- end }}
+    clientConfig:
+      service:
+        namespace: d8-ingress-nginx
+        name: {{ $crd.name }}-admission
+        path: /networking/v1/ingresses
+      caBundle: {{ $context.Values.ingressNginx.internal.admissionCertificate.ca | b64enc }}
+  - name: {{ $crd.name }}.validate.d8-ingress-nginx-deckhouse
+    matchPolicy: Equivalent
+    namespaceSelector:
+      matchExpressions:
+      - key: heritage
+        operator: In
+        values:
+        - deckhouse
+    rules:
+      - apiGroups:
+          - networking.k8s.io
+        apiVersions:
+          - v1
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - ingresses
+        scope: Namespaced
+    failurePolicy: Ignore
+    sideEffects: None
+    timeoutSeconds: 30
```

Comment:
I think we can decrease the timeout here. 30 seconds timeout is needed for huge client's ingresses but for our ingresses we can use 3-5 seconds, especially with Ignore failure policy

Reply:
@yalosev  Maybe at least 10 seconds? I afraid, that 3 seconds can cause false positive even when ingress alive